logging/performance_reliability/logging-flow-control-mechanisms.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-flow-control-mechanisms"]
+= Flow control mechanisms
+:context: logging-flow-control-mechanisms
+
+toc::[]
+
+If logs are being produced faster than they can be collected, it can be difficult to predict or control the volume of logs to an output.
+
+This can result in logs being lost, or in the event of a system outage, when log buffers build up without user control, this can cause long recovery times and high latency when the connection is restored.
+
+As an administrator, you can configure flow control mechanisms for your {logging}. You can choose to limit logging rates, or ignore some logs entirely.
+Logs may still be lost if the collector cannot keep up, but when using flow control mechanisms, you have greater control over which logs are lost, and greater predictability of log rates.
+
+[id="logging-configuring-flow-control-benefits"]
+== Benefits of flow control mechanisms
+
+* The cost and volume of logging can be predicted more accurately in advance.
+* Noisy containers cannot produce unbounded log traffic that downs out other containers.
+* Ignoring logs reduces the load on the logging infrastructure.
+* High-value logs can be preferred over low-value logs by assigning higher rate limits.
+
+[id="logging-configuring-flow-control-about-rate-limits"]
+== Configuring rate limits
```

Comment:
Somehow we need to convey these limits are per collector.  This means the final amount of collected logs is (N x rate) which is not well conveyed in the API

Reply:
Added something to try to explain this, PTAL